### PR TITLE
feature/Search/CLK32-903-SearchScreen

### PR DESCRIPTION
**Pull request by Minhdn1**
## Prerequisite
- [x] Self-review :monocle_face:
- [x] No Crashes ☠️
---

### Root cause
- Ui Screen Home
-Ui Screen Search

### Solution
- update UI Search Screen
- Link sence

### Impact
- Home
- Search
- Register

### Screenshots (Optional)
- Dark

- Light

### CODE REVIEW

**Comment by namnh92** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Big PR</td>
    </tr>
  </tbody>
</table>

- [x] I understand that "Search.Placehodel".localized : "General.Search" is now existed in Localized service

- [x] I understand that "General.Notification" is now existed in Localized service

- [x] I understand that "Home.Profile" is now existed in Localized service

- [x] I understand that "Home.Server" is now existed in Localized service

- [x] I understand that "Login.SignIn" is now existed in Localized service

- [x] I understand that "General.DisplayName" is now existed in Localized service

- [x] I understand that "Register.SignInInstead" is now existed in Localized service

- [x] I understand that "Register.SignUp" is now existed in Localized service

- [x] I understand that "Register.Title" is now existed in Localized service

- [x] I understand that "Search.All" is now existed in Localized service

- [x] I understand that "Search.GroupChat" is now existed in Localized service

- [x] I understand that "Search.Message" is now existed in Localized service

- [x] I understand that "Search.People" is now existed in Localized service

- [x] I understand that "Search.Group" is now existed in Localized service

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: ClearKeep/Scenes/Register/Views/RegisterContentView.swift [hunk 1]
```diff
@@ -28,44 +30,53 @@ struct RegisterContentView: View {
...
-	
+
 	var foregroundColorWhite: Color {
 		AppTheme.shared.colorSet.offWhite
 	}
-	
+
 	var foregroundColorPrimary: Color {
 		AppTheme.shared.colorSet.primaryDefault
 	}
```
Comment (namnh92) [inline ClearKeep/Scenes/Register/Views/RegisterContentView.swift:64]: Don't create new variable if not have any condition.

File: ClearKeep/Scenes/Search/Views/CatalogyView.swift [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+				}
+			}
+		}
+		.padding(.horizontal, Constants.spacing)
+	}
+}
+
+// MARK: - Private Variables
+private extension CatalogyView {
+
```
Comment (namnh92) [inline ClearKeep/Scenes/Search/Views/CatalogyView.swift:52]: Remove break line

File: ClearKeep/Scenes/Search/Views/SearchMessageView.swift [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+
+// MARK: - Private Variables
+private extension SearchMessageView {
+	var foregroundColorUserName: Color {
+		colorScheme == .light ? AppTheme.shared.colorSet.black : AppTheme.shared.colorSet.greyLight
+	}
+	
+	var userImage: Image {
+		AppTheme.shared.imageSet.faceIcon
+	}
```
Comment (namnh92) [inline ClearKeep/Scenes/Search/Views/SearchMessageView.swift:70]: Don't create new variable if not have any condition.

**Review by namnh92 — CHANGES_REQUESTED** [review]
Please update highlight button category

File: ClearKeep/Scenes/Register/Views/RegisterContentView.swift (hunk 1 above)
Comment (Minhdn1) [reply ClearKeep/Scenes/Register/Views/RegisterContentView.swift:64]: fix complete

File: ClearKeep/Scenes/Search/Views/SearchMessageView.swift (hunk 3 above)
Comment (Minhdn1) [reply ClearKeep/Scenes/Search/Views/SearchMessageView.swift:70]: fix complete

**Pull request closed without merging**